docs/src/components/home/Sponsors.tsx

```diff
@@ -0,0 +1,180 @@
+  },
+];
+const GOLDs = [
+  {
+    logo: 'https://github.com/tidelift.png?size=96',
+    name: 'Tidelift',
+    description: 'Enterprise-ready open source software.',
+  },
+  {
+    logo: 'https://github.com/teambit.png?size=96',
+    name: 'Bit',
+    description: 'The fastest way to share code.',
+  },
+  {
+    logo: 'https://images.opencollective.com/callemall/a6946da/logo/96.png',
+    name: 'Text-em-all',
+    description: 'The easy way to message your group.',
+  },
+  {
+    logo: 'https://images.opencollective.com/canadacasino/5b19004/logo/96.png',
+    name: 'Canada Casino',
+    description: 'Safe and rewarding online casino experience',
+  },
+];
+
+function Label({
+  color,
+  children,
+  sx,
+  darker = false,
+}: {
+  color: 'primary' | 'warning';
+  children: React.ReactNode;
+  sx?: BoxProps['sx'];
+  darker?: boolean;
+}) {
+  return (
+    <Box
+      sx={{
+        borderBottomLeftRadius: 10,
+        borderBottomRightRadius: 10,
+        bgcolor: (theme) =>
+          theme.palette.mode === 'dark' ? `${color}.${darker ? '900' : '800'}` : `${color}.50`,
+        color: (theme) =>
+          theme.palette.mode === 'dark' ? `${color}.100` : `${color}.${darker ? '800' : '500'}`,
+        display: 'flex',
+        alignItems: 'center',
+        typography: 'caption',
+        fontWeight: 600,
+        p: 1,
+        ...sx,
+      }}
+    >
+      <Box
+        sx={{
+          display: 'inline-block',
+          mr: 1,
+          borderRadius: 1,
+          width: 8,
+          height: 8,
+          bgcolor: (theme) =>
+            theme.palette.mode === 'dark'
+              ? `${color}.${darker ? '800' : '300'}`
+              : `${color}.${darker ? '800' : '500'}`,
+          border: '2px solid',
+          borderColor: (theme) =>
+            theme.palette.mode === 'dark'
+              ? `${color}.${darker ? '300' : '500'}`
+              : `${color}.${darker ? '300' : '100'}`,
+        }}
+      />
+      {children}
+    </Box>
+  );
+}
+
+const Sponsors = () => {
+  return (
+    <Container sx={{ py: 8 }}>
+      <Typography variant="h2" sx={{ mb: 1 }}>
+        Our sponsors
+      </Typography>
+      <Typography color="text.secondary" sx={{ mb: 4, maxWidth: 450 }}>
+        The continued development and maintenance of Material-UI is greatly helped by our generous
+        sponsors.
+      </Typography>
+      <Grid container spacing={{ xs: 2, md: 4 }} sx={{ mb: 5 }}>
+        {DIAMONDs.map((item) => (
+          <Grid item key={item.name} xs={12} sm={6} md={4}>
+            <Paper
+              variant="outlined"
+              sx={{ height: '100%', display: 'flex', flexDirection: 'column' }}
+            >
+              <Box sx={{ p: 2, display: 'flex', mb: 'auto' }}>
+                <Avatar src={item.logo} alt={`${item.name} logo`} sx={{ borderRadius: '4px' }} />
+                <Box sx={{ ml: 2 }}>
+                  <Typography fontWeight="bold">{item.name}</Typography>
```

Comment:
Reminder to change all typographies with the company name and its description to the variant with 14px. Including the _**Diamond sponsor**_ and _**Gold sponsor**_ texts (they are at 12px now).